src/data/samplers/modeMedian.js

```diff
@@ -0,0 +1,107 @@
+import d3 from 'd3';
+import {identity, noop} from '../../util/fn';
+
+export default function() {
+
+    var undefinedValue = d3.functor(undefined),
+        number = d3.functor(10),
+        accumulator = noop,
+        field = identity;
```

Comment:
Can you call this 'value' so that it matches the property name found in our other calculator components?